sdk/qrcode/android/java/com/google/cardboard/sdk/qrcode/CardboardParamsUtils.java

```diff
@@ -80,6 +80,25 @@
           .appendEncodedPath(URI_PATH_CARDBOARD_HOME)
           .build();
 
+  /**
+   * Converts a bytes array with an URI string to a Uri object.
+   * Obtains the physical parameters of a Cardboard headset from the Uri (as bytes) by
+   * calling {@code createFromUri}.
+   * Then writes the device parameters to a predefined storage location calling {@code
+   * writeDeviceParams()}.
+   *
+   * @param byteArrUri A bytes buffer with the URI to read the parameters from.
+   * @param context The current Context. It is generally an Activity instance or wraps one or an
+   *        Application. It is used to write to storage.
+   */
+  private static void saveParamsFromUri(byte[] byteArrUri, Context context){
+    String strUri = new String(byteArrUri);
```

Comment:
Consider adding some checking in case there is error when trying to convert from array to String and from String to Uri.

Reply:
I think it's important that we check that the bytes[] -> String conversion has been successful if there is a way to do so.